test/go/race_conditions_integration_test.go

```diff
@@ -16,6 +16,14 @@ func setupSingleEntryTable(f *testhelpers.TestFerry, sourceDB, targetDB *sql.DB)
 	testhelpers.SeedInitialData(targetDB, "gftest", "table1", 0)
 }
 
+func addMaintenanceComments(queries []string, comment string) []string {
```

Comment:
Should this be named `addMaintenanceComments`? I think `addAnnotations` or `annotate` may be a better name.